todo_rxbloc_app/lib/base/di/app_dependencies.dart

```diff
@@ -65,13 +66,53 @@ class AppDependencies {
 
   List<SingleChildWidget> get _dataStorages => [];
 
-  List<Provider> get _dataSources => [];
+  List<Provider> get _dataSources => [
+        Provider<TodoService>(
+            create: (context) => TodoService(
+                  ReactiveLocalStorageRepository(
+                    seedValue: [
```

Comment:
What's the issue of adding the initial data as part of the app dependencies ? Do you think we can define such kind of data outside of the app dependencies ?